src/js/Draw/L.PM.Draw.Marker.js

```diff
@@ -194,4 +196,62 @@ Draw.Marker = Draw.extend({
       this._hintMarker?.setIcon(this.options.markerStyle.icon);
     }
   },
+  _setupTouchScreenTips() {
+    let markerHintTip = null;
+
+    this._map.on('pm:drawstart', (event) => {
+      if (event.shape === 'Marker' && !deviceHasFinePointer()) {
+        // Hides the "hint marker" as that is confusing for touch screen users.
+        const drawMarker = this._map.pm.Draw.Marker;
+        drawMarker._hintMarker.remove();
+
+        // Creates the touch screen hint "permanently" (while using the tool)
+        // sticked at the top of the map.
+        markerHintTip = L.tooltip([0, 0], {
+          className: 'leaflet-tooltip-stickynote',
+          content: getTranslation('tooltips.placeMarkerTip'),
+          direction: 'bottom',
+          permanent: true,
+        }).addTo(this._map);
+
+        const updateHandler = () => this._updateHintPosition(markerHintTip);
+
+        updateHandler();
+        this._map.on('zoomlevelschange resize move', updateHandler, this);
+      }
+    });
+    this._map.on('pm:drawend', (event) => {
+      if (event.shape === 'Marker' && markerHintTip) {
+        markerHintTip.remove();
+        markerHintTip = null;
+        this._map.off(
+          'zoomlevelschange resize move',
+          this._updateHintPosition,
+          this
+        );
+      }
+    });
+  },
+  _updateHintPosition(markerHintTip) {
```

Comment:
No need to have `markerHintTip` as argument, if assigned to the context `this._markerHintTip`